react-routing/client/src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Catalog from './pages/Catalog';
+import ProductDetails from './pages/ProductDetails';
+import About from './pages/About';
+import NotFound from './pages/NotFound';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```

Comment:
I don't think any of this HTML is ever rendered. Judging from your screen shots, it is not. Either way, you can delete it from here.